public/css/user.less

```diff
@@ -114,6 +114,19 @@
   font-size: 28px;
 }
 
+.apos-workflow-commit-modal .apos-modal-title {
+  position: relative;
+}
+
+.apos-workflow-commit-modal p.apos-workflow-last-modified {
```

Comment:
does this need to be scoped so hard? it surely doesn't need the element selector?